lib/google/ads/google_ads/autoboxing_fields.rb

```diff
@@ -37,17 +37,19 @@ def self.is_value_field?(class_name)
         def self.patch_constructor_for_autoboxing(fields, repeated_fields, klass_to_patch)
           orig_initialize = klass_to_patch.instance_method(:initialize)
           klass_to_patch.instance_eval do
+            fields_map = fields.map { |field| [field.name.to_sym, field] }.to_h
+            repeated_fields_map = repeated_fields.map { |field| [field.name.to_sym, field] }.to_h
             define_method(:initialize) do |**kwargs|
               new_kwargs = kwargs.dup
-              fields.select { |x| new_kwargs.include?(x.name.to_sym) }.each do |field|
-                value = new_kwargs.fetch(field.name.to_sym)
-                new_kwargs[field.name.to_sym] = AutoboxingMappings.wrapped_mapping(field.subtype.msgclass).call(value)
-              end
-
-              repeated_fields.select { |x| new_kwargs.include?(x.name.to_sym) }.each do |field|
-                value = new_kwargs.fetch(field.name.to_sym)
-                mapping = AutoboxingMappings.wrapped_mapping(field.subtype.msgclass)
-                new_kwargs[field.name.to_sym] = value.map { |x| mapping.call(x) }
+              if !kwargs.empty?
+                kwargs.each do |kwarg, value|
```

Comment:
I noticed you aren't calling `kwarg.to_sym` here, but the old code did. Is that an issue?

Reply:
I don't think we ever called to_sym on the kwarg provided from the user. Those should all be symbols because of how the language works.

We did call to_sym on the field name from the field descriptor, which is still around but done once at the beginning when I generate the map.

Reply:
Sorry, I somehow missed the `to_sym` call above. LGTM.